cross/python313/Makefile

```
@@ -0,0 +1,192 @@
+PKG_NAME = python313
+PKG_VERS = 3.13.0
+PKG_VERS_MAJOR_MINOR = $(word 1,$(subst ., ,$(PKG_VERS))).$(word 2,$(subst ., ,$(PKG_VERS)))
+PKG_EXT = tar.xz
+PKG_DIST_NAME = Python-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://www.python.org/ftp/python/$(PKG_VERS)
+PKG_DIR = Python-$(PKG_VERS)
+
+# Platform "powerpc-none-linux-gnuspe" with compiler "gcc" is not supported by the
+# CPython core team, see https://peps.python.org/pep-0011/ for more information.
+# And compiler must support std=c++11 (OlD_PPC_ARCHS fail, but ARMv5_ARCHS have no issue).
+UNSUPPORTED_ARCHS = $(OLD_PPC_ARCHS)
+
+BUILD_DEPENDS = native/$(PKG_NAME)
```

please avoid variables in DEPENDS definitions (for #6255)
```suggestion
BUILD_DEPENDS = native/python313
```

---

done.